examples/repeatable_test.py

```diff
@@ -0,0 +1,8 @@
+import numpy as np
+
+from fast_binomial import SFC64, Generator
+
+test = np.array([[0, 3, 0], [0, 1, 3]], dtype=int)
+gen = Generator(SFC64(0), 0.5)
+results = gen.binomial(test)
+print(results)
```

Comment:
Is it worth adding a line like this?: `assert Generator(SFC64(123), 0.5).binomial(test).is_equal(Generator(SFC64(123), 0.5).binomial(test))` ?